VERA Programmer's Reference.md

```diff
@@ -302,6 +302,44 @@ By setting the 'Address Increment' field in ADDRx_H, the address will be increme
 
 Setting the **DECR** bit, will decrement instead of increment by the value set by the 'Address Increment' field.
 
+### Video RAM access example
+DATA0/1 registers can get accessed both at the same time enabling efficient screen copy operations such as copy a memory segment, a column or row of text and others. ADDR0_L/M/H is set to the source address and ADDR1_L/M/H to the target address. To copy data read from DATA0 and write to DATA 1. You can loop for the length or do other fancy tricks with it. The increment ammount will give much control on what you want to copy.
+
+To access the three ADDR registers for the correct DATA byte, ADDR_SEL in $9F25 (bit0) need to set accordingly before writing to the ADDRESS registers. 
+Here is a code example to copy 256 bytes from $1000 to $4000. 
+
+```
+begin
+  ;load ADDR0 with $01000 and increment=1
+  LDA #$00   ;load ADDR_SEL for DATA0
```

Comment:
Since this field is shared with DCSEL, I would suggest the following approach in example code:
```
  LDA $9F25  ;load the current value for CTRL
  AND #$FE   ;clear the ADDRSEL bit
  STA $9F25
```